lib/dependencies/DepBlockHelpers.js

```diff
@@ -33,7 +33,7 @@ DepBlockHelpers.getDepBlockPromise = function(depBlock, outputOptions, requestSh
 				"])";
 		}
 	}
-	return "Promise.resolve()";
+	return "new Promise(function(resolve) { resolve(); })";
```

Comment:
@sokra Interesting. Curious as to why is `new Promise(function(resolve) { resolve(); })` better than `Promise.resolve()` here and above?